### ISSUE TITLE
Enhancment for Repo Mangement

### ISSUE THREAD

**Issue by tavallaie** [pgmq/pgmq#224]
Hi, I think we can use this action: [Close Stale Issues](https://github.com/marketplace/actions/close-stale-issues) to clean up issues that haven't had a response for a while. It will also help us use and manage labels more effectively.

**Comment by ChuckHend** [comment]
Good idea. The issues page currently doubles as a feature backlog, so we would want to find a new home for some of these if they are candidates for closing.

**Comment by tavallaie** [comment]
we can use (GithubProject)[https://github.com/tembo-io/pgmq/projects?query=is%3Aopen] for backlog it can automatically close issues and etc. and we can have it Kanban style and etc. 
Also if adding some label to issues can prevent from closing automatically with that action.

**Comment by ChuckHend** [comment]
> Also if adding some label to issues can prevent from closing automatically with that action.

Sounds good to me!

**Comment by tavallaie** [comment]
If you add some label and document to use for them, we can close this issue.

**Comment by ChuckHend** [comment]
> If you add some label and document to use for them, we can close this issue.

What kind of labels beyond what already exists are you thinking would be helpful? If we have some existing issues that are stale, maybe we can write a rule that would identify them as the starting point.

**Comment by tavallaie** [comment]
there are issues that don't have label so if we want setup actions for maybe we need customizing strategy for labels like `stale for 14` and etc.
